### ISSUE TITLE
qsiprep-container on AWS

### ISSUE THREAD

**Issue by arokem** [PennLINC/qsiprep#560]
Hello - this is probably really trivial, but I can't figure it out and I bet (hope) someone else here has experience with this. I am trying to run qsiprep-docker on an AWS instance. I did the following: 

- Start an AWS EC2 instance and login using instance connect (so easy!)
- Update apt packages, use apt to install docker (`apt install docker.io`)
- Install miniconda to the user.
- Create new conda environment and activate it.
- `pip install qsiprep-container`

Now, the problem is that the user account can't run docker because it doesn't have the permissions, but sudo can't see the qsiprep-docker CLI, because it was installed into the user account. I tried setting up a miniconda at the root level, but that only led to other issues, and now I am 🤔 whether this is even possible, or I am just going about this all wrong. Does anyone have any thoughts on this?

**Comment by pcamach2** [comment]
Can you call the qsiprep-docker command from the user's absolute path as sudo?

**Comment by arokem** [comment]
No, when the command is prefixed with `sudo` the terminal no longer sees qsiprep-dockers as installed even when run from the conda environment into which the software was installed, so I get a `sudo: qsiprep-docker: command not found`.

**Comment by arokem** [comment]
BTW: not a show-stopper, by any means. For now, I am just running this directly from docker, and it's happily running along. But just wondering if I am missing a way to make it work.

**Comment by mattcieslak** [comment]
Can you sudo add the user to the docker group?

**Comment by cookpa** [comment]
> Can you sudo add the user to the docker group?

This ought to do it. Here's what I did to get docker running as a user on my machine

https://docs.docker.com/engine/install/linux-postinstall/

**Comment by arokem** [comment]
Thanks @mattcieslak for the idea and @cookpa for the link! Running `sudo usermod -aG docker ubuntu` did the trick! 

I am now realizing that this is more of a discussion than an issue, but I don't think that I can move it to the discussions. Maybe someone with the permissions could migrate it over, rather than close it? Thanks again everyone!
